python/main.py

```diff
@@ -48,25 +50,76 @@ def root():
 @app.post("/items")
 def add_item(name: str = Form(...), category: str = Form(...), image: UploadFile = File(...)):
     logger.info(f"Receive item: {name}, {category}")
-
-    with open(json_file, mode='r') as j:
-        items = json.load(j)
     
+    conn = sqlite3.connect(db)
+    cur = conn.cursor()
+
     image_filename = store_image(image)
     
-    items['items'].append({'name': name, 'category': category, 'image_filename': image_filename})
+    # get category id
+    cur.execute('''SELECT * FROM category
+                WHERE name LIKE ?
+                ''', (category,)
+                )
+    
+    category_record = cur.fetchall()
+ 
+    #[{"category": [{"id": "0", "category": "fashion"}....],},...]
+    if len(category_record) > 0:
+        category_id = int(category_record[0][0])
 
-    with open(json_file, mode='w') as j:
-        json.dump(items, j)
+    else: 
+        cur.execute('SELECT MAX(id) FROM category')
+        max_id = cur.fetchall()[0][0]
+ 
+        if max_id == None:
+            max_id = 0
+        else:
+            max_id = int(max_id)
+        category_id = max_id + 1
         
+        # add record into sqlite
+        cur.execute('INSERT INTO category(id, name) VALUES(?, ?)', (category_id, category))
+
+
+    cur.execute('INSERT INTO items(name, category_id, image_name) VALUES(?, ?, ?)', (name, category_id, image_filename))
+
+    conn.commit()
+
+    # close cur
+    cur.close()
+    # close db
+    conn.close()
+
     return {"message": f"item received: {name}, {category}, {image_filename}"}
 
 
 @app.get("/items")
 def get_items():
-    with open(json_file, mode='r') as getfile:
-        items = json.load(getfile)
-    return items
+    # with open(json_file, mode='r') as getfile:
+    #     items = json.load(getfile)
+    # return items
```

Comment:
Please don't keep the commented-out code, as it decreases readability.
We're keeping history using Git and you can retrieve old code if it becomes necessary.

Reply:
Ok. I erased all of the commented-out code.